### PR TITLE
CEDS-1249 Tests improvements

### PR DESCRIPTION
**Pull request by PatrykRudnicki**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -64,7 +64,8 @@ lazy val scoverageSettings: Seq[Setting[_]] = Seq(
     "metrics\\..*",
     "features\\..*",
     "test\\..*",
-    ".*(BuildInfo|Routes|Options|TestingUtilitiesController).*"
+    ".*(BuildInfo|Routes|Options|TestingUtilitiesController).*",
+    "logger.*\\(.*\\)"
```
Comment (PatrykRudnicki) [inline build.sbt]: I tried to exclude logs from coverage, this line exclude just `logger.error(` If someone know how to make it to exclude message too please ping me

File: app/controllers/ChoiceController.scala [hunk 2]
```diff
@@ -88,8 +88,6 @@ class ChoiceController @Inject()(
                   Redirect(controllers.routes.CancelDeclarationController.displayForm())
                 case Submissions =>
                   Redirect(controllers.routes.SubmissionsController.displayListOfSubmissions())
-                case _ =>
-                  Redirect(controllers.routes.ChoiceController.displayPage())
```
Comment (PatrykRudnicki) [inline app/controllers/ChoiceController.scala]: There is no possibility to achieve this case. Form validation will block this. Should we stay with this or removing is okay?

File: build.sbt (hunk 1 above)
Comment (mattclark-zerogravit) [reply build.sbt]: nice! 👍

File: app/controllers/ChoiceController.scala (hunk 2 above)
Comment (mattclark-zerogravit) [reply app/controllers/ChoiceController.scala]: Will we get a warning if it is removed? if so can we suppress it?

Comment (PatrykRudnicki) [reply app/controllers/ChoiceController.scala]: No, there is no warning

**Pull request merged**